### ISSUE TITLE
Tab keybinding does not work

### ISSUE THREAD

**Issue by mturquette** [astroidmail/astroid#479]
I have the following in my ~/.config/astroid/keybindings:

`main_window.next_page=Tab`

But instead of cycling through the windows, pressing the `tab` key just selects the random buttons on the interface as you would expect it to in most programs. You can see what I mean in the two screenshots below. The first image is how things look before pressing tab, the second is after pressing tab:

![beforetab](https://user-images.githubusercontent.com/13019/37238848-249424ea-23e4-11e8-8fab-ba5473bb76c0.png)

![aftertab](https://user-images.githubusercontent.com/13019/37238847-2474cabe-23e4-11e8-9ba2-df99aec9fae5.png)

I have also tried `KP_Tab` and `ISO_Left_Tab`. Binding it to a "normal" unused key such as `i` yields the desired behavior. Checking the log I see that there is no Error from mapping the Tab key.

OS is Mac OS X High Sierra 10.13.3

Any thoughts?

**Comment by gauteh** [comment]
Does Tab normally work (before you bind it to something else) like it should to jump to next unread?

**Comment by gauteh** [comment]
I noticed that Tab cannot be defined for main_window, but can be defined for the sub-modes. Probably because the notebook (pages) catches the event before it gets to the main window.

**Comment by mturquette** [comment]
Correct, the default binding (jump to next unread) works fine.

**Comment by mturquette** [comment]
Ok, so does that close the issue? I've simple bound another key for now

**Comment by gauteh** [comment]
Not really, but if it does not bug anyone let’s close it for now. Enough
other issues 😉

lør. 10. mar. 2018 kl. 17:33 skrev Michael Turquette <
notifications@github.com>:

> Ok, so does that close the issue? I've simple bound another key for now
>
